2019-12-09 17:51:13,296 manager INFO 1.1.0
2019-12-09 17:51:13,296 manager INFO Copyright (C) 2003-2010
2019-12-09 17:51:13,296 manager INFO   Noriaki Ando
2019-12-09 17:51:13,296 manager INFO   Intelligent Systems Research Institute, AIST
2019-12-09 17:51:13,296 manager INFO Manager starting.
2019-12-09 17:51:13,296 manager INFO Starting local logging.
2019-12-09 17:51:13,302 manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2019-12-09 17:51:15,309 ManagerServant INFO Master manager not found
2019-12-09 17:51:15,312 manager WARNING Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\OpenRTM_aist\Manager.py", line 1900, in initManagerServant
    reffile = file(self._config.getProperty("manager.refstring_path"),'w')
IOError: [Errno 2] No such file or directory: '/var/log/rtcmanager.ref'

2019-12-09 17:51:15,313 LocalServiceAdmin INFO All the local services are enabled.
2019-12-09 17:51:15,328 ec_base INFO setRate(1000000.000000) done
2019-12-09 17:51:15,328 ec_base INFO setRate(1000.000000) done
2019-12-09 17:51:15,329 rtobject INFO 1 execution context was created.
2019-12-09 17:51:15,336 rtobject INFO Initial active configuration set is default.
2019-12-09 17:51:18,372 SdoServiceAdmin WARNING No available SDO service in the factory: 
2019-12-09 17:51:18,374 SdoServiceAdmin ERROR type  already exists.
2019-12-09 17:51:18,374 SdoServiceAdmin WARNING No available SDO service in the factory: IDL:OpenRTM/ComponentObserver:1.0
2019-12-09 17:51:18,374 SdoServiceAdmin ERROR type 6ed13448-4e3f-426f-a023-58c0a1438a06 already exists.
